sirepo/template/omega.py

```diff
@@ -126,6 +128,33 @@ def background_percent_complete(report, run_dir, is_running):
     )
 
 
+def copy_related_sims(data, qcall=None):
+    for index, sim_obj in enumerate(data.models.simWorkflow.coupledSims):
+        if sim_obj.simulationType and sim_obj.simulationId:
+            p = pkio.py_path(
+                simulation_db.find_global_simulation(
```

Comment:
You probably want checked=True.

Reply:
I have a related issue that I can add `checked=True`. Thank you for pointing out